src/utilities/utility.ts

```diff
@@ -129,6 +137,75 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
     return commandExecutionResults;
 }
 
+export async function getFilePathsConfigs(): Promise<FileConfigPath> {
+
+    let filePathsConfig = <FileConfigPath>{};
+    const MANIFEST_PATHS_KEY = 'manifestFilePaths';
+    const HELM_CHART_KEY = 'helmChartFilePaths';
+    const DOCKERFILE_PATH_KEY = 'dockerfilePaths';
+    const DOCKERFILE_PATH_LABEL_KEY = 'dockerfile-path';
+
+    let inputManifestFiles = inputParams.manifests || [];
+    filePathsConfig[MANIFEST_PATHS_KEY] = inputManifestFiles;
+
+    let helmChartPaths = (process.env.HELM_CHART_PATHS && process.env.HELM_CHART_PATHS.split('\n').filter(path => path != "")) || [];
+    filePathsConfig[HELM_CHART_KEY] = helmChartPaths;
+
+    let imageNames = core.getInput('images').split('\n');
+    let imageDockerfilePathMap: any = {};
+    let pathValue: string, pathLink: string;
+    const branchOrTag: string = process.env.GITHUB_REF && process.env.GITHUB_REF.replace('refs/heads/','blob/').replace('refs/tags/','blob/');

```

Comment:
What value does `blob/` do? is it for creating a path to a particular file? If so, you should hard-code the constant where you're generating it and let the `branchOrTag` be set to the particular name. 

Also consider calling the variable `reference` maybe? It seems a little generic

Reply:
The path to the dockerfile shows up as :
https://github.com/<Username/RepoName>/blob/<branchOrTagName>/<DockerfileName>
The value is either the branch or tag name , hence the naming. 

`blob/` is part of the filepath. Is there another better way to get the filepath that I missed ?

Reply:
Addressed it in a comment below

Reply:
ok, resolving this.